samples/meetings-content-bubble/csharp/TargetedNotifications/Pages/SendNotificationPage.cshtml

```diff
@@ -0,0 +1,30 @@
+﻿@page
+@model TargetedInNotificationMeetingBot.Pages.SendNotificationPage
+@{
+    ViewData["Title"] = "Layout";
+}
+
+<script>
+    microsoftTeams.app.initialize();
+    function validateForm() {
+        var feedbackInfo = {
+            Topic: 'Meeting Targeted Notification',
```

Comment:
Minor: Targeted Meeting Notification or Targeted In-Meeting Notification

Reply:
Changed this